compose/mpp/demo/src/wasmJsMain/resources/index.html

```diff
@@ -18,24 +18,13 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
`<meta name="viewport" content="width=device-width, initial-scale=1.0">` - better to keep it here too